src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -68,6 +70,7 @@ object AttemptedLookupResult {
  * Trait to be implemented by concrete backends.
  */
 trait Backend {
+
```

Comment:
Phew.  I'm not sure what the rules on whitespace are here, I was worried I might be on the receiving end of an entirely different hand gesture.